### DISCUSSION TITLE
Mangal 4.0.7 Release date?

### DISCUSSION THREAD

**Discussion by lramos15** [metafates/mangal#182] [Q&A]
Last commit appears to be from February, but the released build is from January 13th. Can we get another release for 4.0.7 please :). I want to use the new feature of disabling `_` in favor of ` `

**Comment by metafates** [comment]
Hi!

The next release will be tagged as 5.0.0. And it will support flexible filename templates. So... I don't think 4.0.7 will be a thing. Speaking of release date - I don't know, sorry! I'm currently working on it as much as I can in my free time.
